tests/conftest.py

```diff
@@ -222,7 +222,7 @@ def shutdown(self):
 @pytest.fixture
 def hlwm(hlwm_process):
     display = os.environ['DISPLAY']
-    #display = ':13'
+    # display = ':13'
```

Comment:
Does this serve any purpose, @t-wissmann? Maybe just remove it...

Reply:
(same goes for the similar line further below)

Reply:
I sometimes activate this lines to let the tests run in a `xephyr` (instead of a `xvfb`).

Reply:
I'll keep it then

Reply:
FWIW it's quite straightforward to [add custom commandline options](https://docs.pytest.org/en/latest/example/simple.html) to pytest for your testsuite (basically, add a `pytest_addoption` hook adding the option, and here use the `config` fixture and check `config.getoption('--xephyr')` or so).